src/utils/ohri-form-helper.ts

```diff
@@ -1,27 +1,57 @@
 import dayjs from 'dayjs';
-import { LayoutType } from '@openmrs/esm-framework';
+import { LayoutType, showToast } from '@openmrs/esm-framework';
 import { fetchConceptNameByUuid } from '../api/api';
 import { ConceptTrue } from '../constants';
 import { EncounterContext } from '../ohri-form-context';
-import { OHRIFormField, OHRIFormPage, OHRIFormSection, SubmissionHandler } from '../api/types';
+import { OHRIFormField, OHRIFormPage, OHRIFormSection, OpenmrsEncounter, SubmissionHandler } from '../api/types';
 import { OHRIDefaultFieldValueValidator } from '../validators/default-value-validator';
 import { isEmpty } from '../validators/ohri-form-validator';
 import { isTrue } from './boolean-utils';
 
+export const validateLNDbirthCount = (encounter: OpenmrsEncounter) => {
+  if (!encounter) {
+    const errorMessage = 'Missing encounter';
+    console.error(errorMessage);
+    return errorMessage;
+  }
+
+  const { obs: obsArray } = encounter;
+  const birthInfoObs = obsArray.filter((eachObs) => eachObs.concept === '1c70c490-cafa-4c95-9fdd-a30b62bb78b8');
+  const [birthCount] = obsArray.filter((eachObs) => eachObs.formFieldPath === 'ohri-forms-birth_count');
```

Comment:
FE shouldn't know about concepts and form field paths as these are bound to be implementation specific

Reply:
Noted.